testsuite/features/build_validation/smoke_tests/smoke_tests.template

```diff
@@ -43,7 +43,6 @@ Feature: Smoke tests for <client>
     And I click on "Apply Patches"
     And I click on "Confirm"
     Then I should see a "1 patch update has been scheduled for" text
-    When I force picking pending events on "<client>" if necessary
```

Comment:
since, in the BV testsuite, it looks like client is being used generically, do we need to remove this particular line as well? or was it only necessary for traditional clients? I assume this because nothing else got changed for the BV "<client>"